### DISCUSSION TITLE
[Unearth] Model displaying differently in Design Review vs. the embedded iTwin Viewer

### DISCUSSION THREAD

**Discussion by gabe-unearth** [iTwin/itwinjs-core#2718] [Q&A] [answered]
The model in the embedded viewer has extra content that is not visible in the Design Review viewer in the Connect portal.  See "diff" attachment for reference.  On the left, is the viewer that is embedded in our app. Is this expected?
![iModel viewer diff](https://user-images.githubusercontent.com/62312126/142039115-ddf3a0b9-9695-4b5c-935c-117c75f98eab.png)

**Comment by pmconne** [comment]
In Design Review, did you open a saved view, or did you use the "3d Models" page to select a set of models to view?
In Design Review, if you switch from the Model Tree to the Category Tree, are all the categories enabled; and if not, does enabling them all cause the missing content to be displayed?
If the iModel has more than one version, did you open the same version in both apps?

**Reply by gabe-unearth** [reply]
1. Opened directly from "3D models".
2. All categories are enabled in Design Review.
3. We're currently only importing the latest version of the model. Verified that no other (previous) version of the same model in Design Review looks like it does in the embedded viewer. 

Thanks!

**Comment by pmconne** [comment, marked as answer]
In Design Review there's a toggle for "Constructions" that is turned off by default. What happens if you toggle it on?
![image](https://user-images.githubusercontent.com/22944042/142067060-170c5651-dd84-4ccc-a9c0-920a1ad5efc9.png)
